CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
We need to include some email addresses for people to reach out to in here ideally. This could be foss@, but ideally something more concrete.

Reply:
It's buried down under "enforcement", but when you make a CoC through GitHub they ask for an email, and I choose that one.

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at foss@circleci.com.

Reply:
Oh cool +1, I missed it

Reply:
@DanToml If that is not obvious enough, can we open a new PR to make it so?